vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction_test.go

```diff
@@ -321,12 +322,13 @@ func TestEvictReplicatedByReplicaSet(t *testing.T) {
 		assert.True(t, eviction.CanEvict(pod))
 	}
 
+	basicVpa := getBasicVpa()
```

Comment:
Can we move it above line 319? That way, we can avoid calling this function again:
```
factory.NewPodsEvictionRestriction(pods, basicVpa)
```